tests/e2e/common.go

```diff
@@ -44,9 +44,7 @@ var (
 	dynamicClient       dynamic.DynamicClient
 	testEnv             *envtest.Environment
 	ctx                 context.Context
-	cancel              context.CancelFunc
-	suiteTimeout        = 35 * time.Minute // Note: if we start seeing "client rate limiter: context deadline exceeded", we need to increase this value
```

Comment:
I removed `suiteTimeout` and the timeout that was in Github Actions for the e2e test. Now there's just the one timeout in the Makefile target for `go test` itself. I think that should be sufficient.